### ISSUE TITLE
when using cellAttr function doesn't respect formatted title

### ISSUE THREAD

**Issue by aulea** [free-jqgrid/jqGrid#441]
Forked one your JSFiddle demo to describe it better: https://jsfiddle.net/otpmf9h4/6/

Date field's title respects normally the given date formatter. But when using it together with cellAttr function (and in cellAttr don't describe/override title), then formatting is lost.

Maybe related to already reported issue: https://github.com/free-jqgrid/jqGrid/issues/439

Will try to provide pull-request to fix it, cause i have already located the bug.
https://github.com/free-jqgrid/jqGrid/blob/master/js/jquery.jqgrid.src.js#L3612
There should be used tv argument.

**Comment by OlegKi** [comment]
Sorry, but I'd ask you **to describe** the problem more exactly before posting pull request. What exactly demonstrates the demo https://jsfiddle.net/otpmf9h4/6/? How one could reproduce the problem (please include step by step instruction to reproduce the problem)? What is expected behavior and what one can see instead? It could be that you want to display unformated data as the tooltip for the `date_issue` column? In the case one can use existing `useUnformattedDataForCellAttr: false` option. See modified demo https://jsfiddle.net/OlegKi/otpmf9h4/17/. The option `useUnformattedDataForCellAttr: false` option changes additionally the value of `cellValue` forwarded to `cellattr` callback from the source data value to **formatted value**. See https://jsfiddle.net/OlegKi/otpmf9h4/27/, which generated custom `title` which contains formatted cell value.

About the pull request. You included `titleValue = stripHtml(tv)` initialization, which **strips** some characters, which could be important to display in the tooltip. Such changes breaks the current behavior of free jqGrid. Look at the demo https://jsfiddle.net/OlegKi/otpmf9h4/16/ and examine tooltips displayed over the texts `test's` and `test "2"`. The current code displays the same texts as tootip, but adding `titleValue = stripHtml(tv)` initialization will break the behavior. Because of that I can't commit your changes.

**Comment by aulea** [comment]
Yes, it does the trick. And when still needed, then from rdata its possible to get still unformatted value as well.
